examples/projections/conic/conic_albers.py

```diff
@@ -2,8 +2,20 @@
 Albers Conic Equal Area
 =======================
 
+This projection, developed by Heinrich C. Albers in 1805, is predominantly used to map
+regions of large east-west extent, in particular the United States. It is a conic,
+equal-area projection, in which parallels are unequally spaced arcs of concentric
+circles, more closely spaced at the north and south edges of the map. Meridians, on the
+other hand, are equally spaced radii about a common center, and cut the parallels at
+right angles. Distortion in scale and shape vanishes along the two standard parallels.
+Between them, the scale along parallels is too small; beyond them it is too large.
+The opposite is true for the scale along meridians.
+
 ``Blon0/lat0/lat1/lat2/width``: Give projection center ``lon0/lat0`` and two standard
 parallels ``lat1/lat2``.
+
+For further information, see
+:gmt-docs:`cookbook/map-projections.html#albers-conic-equal-area-projection-jb-jb`
```

Comment:
I think we can remove these long links to the GMT cookbook.

Reply:
Removed.